app/views/users/show.html.erb

```diff
@@ -1 +1,13 @@
-<h1>This is show user detail page</h1>
\ No newline at end of file
+<%= render 'shared/user', user: @user%>
+<div class='user-bio'> Bio<br><%= @user.bio %></div>
+<ul class='user-card-post-ul'>
+<% if @user.recent_posts.empty? %>
+  <p class='post-card-no-click'>No posts</p>
+<% else %>
+<%@user.recent_posts.each do |post|%>
+<li>
+  <%=render 'shared/post', post: post%>
+</li>
+<%end%>
+<%end%>
+<div class='see-all-post-btn-container'><%= link_to "See all posts", user_posts_path(@user), class: 'see-all-post-btn' %></div>
```

Comment:
- [ ] Also kindly ensure that the user show page follows and closely implements the wireframe guidelines as shown below 👇 
### Wireframe
![image](https://github.com/BranBayou/ror-blog-app/assets/46839548/30cf5aca-94c2-49c0-8e48-78052d6413ee)

### Your Implementation
![image](https://github.com/BranBayou/ror-blog-app/assets/46839548/cf89c241-3b03-489a-9807-34aee5a58ce9)